_maps/shuttles/whiteship_birdshot.dmm

```diff
@@ -1084,15 +1083,6 @@
 	},
 /turf/open/floor/iron/small,
 /area/shuttle/abandoned/pod)
-"Jk" = (
-/obj/effect/decal/cleanable/dirt,
-/obj/machinery/power/apc/auto_name/directional/south,
-/obj/structure/cable,
-/obj/machinery/atmospherics/pipe/smart/manifold4w/supply/hidden{
-	dir = 4
-	},
-/turf/open/floor/iron/smooth_large,
-/area/shuttle/abandoned/cargo)
```

Comment:
This looks like you deleted an APC and smartpipe on the whiteship.

Reply:
Correct, I did delete an apc, because there were 2 in one area. See the mapdiffbot.

Also the smartpipe is still there, due to how the tgm format works it just used another template instead, the `Ui` one

Reply:
Posting it here for clarity
```
"Ui" = (
/obj/effect/decal/cleanable/dirt,
/obj/structure/cable,
/obj/machinery/atmospherics/pipe/smart/manifold4w/supply/hidden{
	dir = 4
	},
/turf/open/floor/iron/smooth_large,
/area/shuttle/abandoned/cargo)
```